scripts/variantstore/docs/aou/vds/cluster/AoU Delta VDS Cluster Configuration.md

```diff
@@ -1,6 +1,6 @@
-# Delta VDS Hail Cluster Configuration
+# VDS Hail Cluster Configuration
 
-To work with a Delta-sized VDS (roughly 250K samples) we use our PMI ops accounts to create a cluster in a Terra
+To work with a AoU-callset-sized VDS (roughly 250K samples) we use our PMI ops accounts to create a cluster in a Terra
```

Comment:
Should this be now 'roughly 400K samples'? (or too soon?)

Reply:
I think I might be a bit too superstitious....